### ISSUE TITLE
Improvements for Why fork Drupal page

### ISSUE THREAD

**Issue by oadaeh** [backdrop-ops/backdropcms.org#447]
The Why fork Drupal? page (at https://backdropcms.org/node/228) needs at least two improvements:

1. It needs a friendly URL.
2. The link for "our principles" needs to be fixed or removed. It currently links to https://backdropcms.org/principles which does not exist.

Also, I found this page via a link at the bottom of https://backdropcms.org/philosophy

**Comment by klonos** [comment]
1. https://backdropcms.org/why-fork-drupal already exists and links to that node. Both the "Why fork Drupal?" links in https://backdropcms.org/philosophy point to that friendly URL. Curious how you happen to come to then `/node/` one? ...unless someone fixed things before I got to this 😄 

2. I have fixed that link to point to `/philosophy` https://backdropcms.org/philosophy#principles and have also added a URL redirect from `/principles` `/philosophy#principles`.
